types/openlayers/index.d.ts

```diff
@@ -13581,6 +13597,7 @@ declare module olx {
          *     zIndex: (number|undefined)}}
          */
         interface VectorOptions {
+            renderMode?: (olx.layer.VectorRenderType | string);
```

Comment:
This definition does not match v4.3.4. Either use the definition from v4.3.4 (https://openlayers.org/en/v4.3.4/apidoc/ol.layer.Vector.html) or change the version at the top of the file to match the version where this change appeared (its seams to be v4.6.0).

Reply:
Thank you for reviewing.

I did update to v4.6.0. But going through the [changelog](https://github.com/openlayers/openlayers/blob/ca3e11a10b47ba0323b3fbc477ce1d39eb06b26e/changelog/v4.6.0.md), I realized that more work was needed to align to this specific version. And it's still behind the current version that comes from npm (v4.6.4). On one hand I really need the performance boost that comes from this option on layer.Vector, but on the other one I don't feel comfortable bringing a version mismatch on other things, even though it worked fine till now. I could dedicate another couple of hours in the next 24 hours to update other parts of the definitions, but I've got no idea if it will be enough. And this PR would not be relevant anymore by then.

What do you think?

Reply:
It's good for me. It has always work that way. If some parts don't match the new version, some people will fix according to their need.